### PR TITLE
Make image form field required.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

File: marketpulse/main/templates/media_new.html [hunk 1]
```diff
@@ -54,12 +52,23 @@ <h2 class="title">New Photo</h2>
...
             </div>
             <input type="file" name="image" id="form-image">
+            <div id="image-errors">
+              {{ field_errors(contribution_form.image) }}
+            </div>
+          </div>
+        </fieldset>
+        <fieldset>
+          <legend>Comment</legend>
+          <div class="form-group">
```
Comment (akatsoulas) [inline marketpulse/main/templates/media_new.html]: Why are you adding another comment form here?

Comment (johngian) [inline marketpulse/main/templates/media_new.html]: Bad rebase :\

File: marketpulse/main/templates/media_new.html [hunk 2]
```diff
@@ -54,12 +52,23 @@ <h2 class="title">New Photo</h2>
...
           <legend>Media</legend>
           <div class="form-group">
-
             <div id="upload-container"></div>
             <div>
               <a href="#" class="btn btn-default btn-block" id="pickimage" data-role="button">Pick Image</a>
             </div>
             <input type="file" name="image" id="form-image">
+            <div id="image-errors">
+              {{ field_errors(contribution_form.image) }}
```
Comment (akatsoulas) [inline marketpulse/main/templates/media_new.html:62]: The error message  is on top of the `Pick Image` button.

**Comment by akatsoulas** [comment]
r+wc :coffee:

**Pull request merged**